external_search.py

```diff
@@ -1738,6 +1738,8 @@ def from_worklist(cls, _db, worklist, facets):
         target_age = inherit_one('target_age')
         collections = inherit_one('collection_ids') or library
 
+        license_datasource_id = inherit_one('license_datasource_id')
+
```

Comment:
I discovered a circulation feature I'd forgotten about where you can get a feed of the books provided by a specific license source. We already index this information in ElasticSearch (in order to filter out undeliverable audiobooks) so it was easy to add this capability to `Filter`.